app/controllers/stories_controller.rb

```diff
@@ -220,7 +220,7 @@ def handle_article_show
   end
 
   def assign_feed_stories
-    feed = Articles::Feed.new(page: @page, tag: params[:tag])
+    feed = Articles::Feed.new(user: current_user, page: @page, tag: params[:tag])
```

Comment:
This path will be cached at the edge so shouldn't pass `current_user`.

Reply:
Are you saying I can't have access to current_user inside `Articles::Feed`?

Reply:
You _can_ have access to `current_user` inside `Articles::Feed`, but within `stories_controller` it's only going to be serving cached views.

But the version of `Articles::Feed` that is getting fetched through `feeds_controller` have all the `current_user` you can eat.

I noticed you _didn't_ add `current_user` to the version of `assign_feed_stories` in the `feeds_controller`.

Unless I'm mistaken, maybe you put it in the wrong place?

Reply:
Hmm: I think you may be write. I want to rework this a little bit: I’m mixing up `user_logged_in?` and passing `current_user` and it’s feels bad!